wscript

```diff
@@ -54,6 +54,7 @@ class Subproject:
 		return True
 
 SUBDIRS = [
+	Subproject('3rdparty/opus'),
```

Comment:
Probably better be `mandatory = True`? For now, this code doesn't seem to have alternative code path for no voice support anyway.